packages/circuit-ui/components/Body/__snapshots__/Body.spec.tsx.snap

```diff
@@ -3,9 +3,9 @@
 exports[`Body should render as a "alert" variant 1`] = `
 .circuit-0 {
   font-weight: 400;
-  margin-bottom: 16px;
   font-size: 16px;
   line-height: 24px;
+  margin-bottom: 16px;
```

Comment:
Looks like we need to add the `noMargin` props in the tests.

Reply:
Just to be on the clear side. You mean to update the tests with noMargin props in all of the components so far. 
We were just thinking it might be a big of a change to update all of them right now, and maybe to do so once we remove the noMargin completely? Wdyt?

Reply:
Yes, that's what I mean. I would do it now — we should treat `noMargin` as the default since that's how developers are expected to treat it.

Reply:
Okies, thanks for clearing it up. Then I'm off to updating all of them.

Reply:
Only issue with this approach is that we'll have to remove noMargin everywhere later (since that's our migration strategy). But sure, I don't mind much 🤷 (and it will be codemodded)